lib/media/track/localaudiotrack.js

```diff
@@ -264,18 +269,25 @@ class LocalAudioTrack extends LocalMediaAudioTrack {
 }
 
 /**
- * The {@link LocalAudioTrack} was disabled, i.e. "muted".
```

Comment:
Same as above comments about renaming "muted" to "paused" and "unmuted" to "play".